spec/controllers/announcements_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+# rubocop:disable Metrics/BlockLength
+RSpec.describe AnnouncementsController, type: :controller do
+  let(:episode) { create(:episode) }
+  let(:announcement) { create(:announcement, episode: episode) }
+
+  context 'with authorized user' do
+    let(:role) { create(:role) }
+    before do
+      @user = create(:user, role: role)
+
+      allow(controller).to receive(:authenticate_user!).and_return(true)
+      allow(controller).to receive(:current_user).and_return(@user)
+    end
+
+    after do
+      @user.delete
+    end
+
+    it 'show announcements #index to autorized user' do
+      get :index, params: { episode_id: episode.id }
```

Comment:
`get:index, epesode_id: episode.id`

Reply:
it doesn't work: `ArgumentError: unknown keywords: id, episode_id`